.eslintrc.json

```diff
@@ -18,14 +18,18 @@
         "react/prop-types": "off",
         "react/react-in-jsx-scope": "off",
         "react/display-name": "off",
+        "prefer-const": "warn",
```

Comment:
I think we should open a ticket to turn these rules back on when we have the time -- they seem useful to me, and in the long term I think should be kept on unless we can make a strong argument that they are not necessary.

Reply:
A review of them would not be terrible. But they haven't been disabled carelessly or without thought.

As far as I know, I'm the only developer on this project that uses typescript. React/prop-types without typescript is an archaic rule; you'll never really see explicit proptypes definitions beyond like React 15. Without typescript, this rule requires explicit proptypes.

React-in-jsx-scope is only relevant to projects running React <17.

Prefer-const is generally a good rule. I think I had to set it to warn since a lot of existing code violates it. It could maybe be turned back on.

React/display-name is only for debugging.